src/App.tsx

```diff
@@ -18,186 +85,98 @@ export const App: React.FC = () => {
           {/* this button should have `active` class only if all todos are completed */}
           <button
             type="button"
-            className="todoapp__toggle-all active"
+            className={cn('todoapp__toggle-all', {
+              active: todos.every(todo => todo.completed === true),
```

Comment:
```suggestion
              active: todos.every(todo => todo.completed),
```

And also create variable for this